src/Robot.Drivers/Sonar/HcSr04DistanceSensorDriverSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Robot.Drivers.Sonar
+{
+    public class HcSr04DistanceSensorDriverSettings
+    {
+        /// <summary>
+        /// Unique ID to refer this sensor.
+        /// </summary>
+        public int SensorId { get; set; }
+
+        /// <summary>
+        /// Sensor angle in radians.
+        /// </summary>
+        /// <remarks>Zero is a forward direction of a Robot. Pi/2 is a right side, -Pi/2 is a left side</remarks>
```

Comment:
Is there any reason why using radians instead of degrees? (only asking because my brain thinks in degrees :) )

Reply:
Good question! My brain thinks in degrees too. However, it's easier to use radians because all the trigonometric functions think in radians (as function arguments and results). This way we may avoid unnecessary transformations.